doc/nrf/create_application.rst

```diff
@@ -0,0 +1,224 @@
+      #. Click :guilabel:`Create Application`.
+
+      See the `extension documentation <Create a new application_>`_ for more information.
+
+You can now start :ref:`configuring and building <configuration_and_build>` the application.
+
+.. _creating_cmd:
+
+Creating application for use with command line
+==============================================
+
+The recommended process for command line takes advantage of Nordic Semiconductor's example application repository, similar to the example application used for :ref:`creating an application in Zephyr <zephyr:application>`.
+
+.. include:: /releases_and_maturity/developing/adding_code.rst
+   :start-after: example_app_start
+   :end-before: example_app_end
+
+.. note::
+   Nordic Semiconductor recommends using the example application repository, but you can also create the application by hand.
+   Follow the steps to :ref:`create your own application by hand <zephyr:application>` in Zephyr and :ref:`create your own manifest repository <dm_workflow_4_own_repo>` in the |NCS|.
+
+Use the following steps depending on the application type:
+
+.. tabs::
+
+   .. group-tab:: Workspace application (recommended)
+
+      To create a workspace application:
+
+      1. Open the `ncs-example-application`_ repository in your browser.
+      #. Click the :guilabel:`Use this template` button on the GitHub web user interface.
+         This creates your own copy of the template repository.
+         In the copy of the repository, the :file:`app` directory contains the template application that you can start modifying.
+      #. Open a command line terminal.
+      #. Initialize the repository with the repository name and path you have chosen for your manifest repository (*your-name/your-application* and *your-app-workspace*, respectively).
+         *your-app-workspace* corresponds to :file:`ncs/` in the :ref:`workspace application structure <create_application_types_workspace>`.
+         Use the following command pattern:
+
+         .. parsed-literal::
+            :class: highlight
+
+            west init -m https:\ //github.com/*your-name/your-application* *your-app-worskpace*
+
+      #. Go to the *your-app-workspace* directory using the following command pattern:
+
+         .. parsed-literal::
+            :class: highlight
+
+            cd *your-app-workspace*
+
+      #. Run the following west command to download the contents of the |NCS|:
+
+         .. code-block::
+            :class: highlight
+
+            west update
+
+         west will clone the |NCS| contents next to the example application directory.
+
+      For more information, see the detailed description of the :ref:`workspace application workflow <dm_workflow_4>`.
+
+   .. group-tab:: Freestanding application
+
+      To create a freestanding application:
+
+      1. Open the `ncs-example-application`_ repository in your browser.
```

Comment:
why are we mixing freestanding app with example-application? I think we should not mix both, as example-application is a workspace template, even if it can be later used to build other freestanding apps.

Reply:
I agree we shouldn't mix them and that this is not an optimal solution. From what I understand, this is the only way to get to create a freestanding application outside of the VSC extension and using the recommended approach with the example-application.

Reply:
What I mean is that example-application should likely be used only to demonstrate a workspace based solution. freestanding should use the "standard NCS workspace".

Reply:
What do you mean by "use the standard NCS workspace"? 
A) This approach: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/developing/adding_code.html#workflow-2-out-of-tree-application-repository ?
B) Or this approach: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/developing/adding_code.html#creating-your-own-manifest-repository-from-scratch

Reply:
Applied edits.